maquetado/views/login.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="../application.css">
+</head>
+<body>
+  <div class="login-container">
+    <div class="login-box">
+        <img class="logo-img" src="../assets/Logo-wolox.png" alt="logo wolox">
+        <label class="input-label" >Email</label>
+        <input class="input-data" type="text" name="email">
+        <label class="input-label">Password</label>
+        <input class="input-data" type="password" name="password">
+        <button class="login-btn"><a href="books-list.html">Login</a></button>
```

Comment:
there is no need to put an <a> tag inside the button. You can do:

`<a class="login-btn" href="books-list.html">Login</a>`